src/ripple/app/tx/impl/apply.cpp

```diff
@@ -115,6 +115,7 @@ apply (Application& app, OpenView& view,
     STTx const& tx, ApplyFlags flags,
         beast::Journal j)
 {
+    disableUnderflowFix(view.info().parentCloseTime);
```

Comment:
There's not much value to that to be honest, after the job thread finishes applying the transaction it immediately becomes free for another operation.